iOS CLI build fails during code signing

I'm trying to run `make ipackage` or `make iframework`, but getting this:

```
mkdir -p build/ios
(cd build/ios && cmake -G Xcode ../.. \
        -DCMAKE_TOOLCHAIN_FILE=../../platform/ios/toolchain.cmake \
        -DMBGL_PLATFORM=ios)
-- Configuring done
-- Generating done
-- Build files have been written to: /Users/kkaefer/Code/gl/native/build/ios
FORMAT= BUILD_DEVICE= SYMBOLS= \
    ./platform/ios/scripts/package.sh
Configuring dynamic and static Release framework for iphoneos; symbols: YES; self-contained static framework: NO
* Recording library version…
mapbox-gl-native b32ba799e0
* Building targets (build 9259, version 3.4.0-beta.2)…
▸ Building ios/settings [Release]
▸ Check Dependencies
▸ Copying Root.strings
▸ Copying Root.plist
▸ Touching Settings.bundle
▸ Signing /Users/kkaefer/Code/gl/native/build/ios/Release-iphonesimulator/Settings.bundle
▸ Building ios/bundle [Release]
▸ Check Dependencies
▸ Processing Info-static.plist
▸ Copying Localizable.strings
▸ Check Dependencies
▸ Running script 'CMake Rules'
** BUILD FAILED **


The following build commands failed:
    CodeSign /Users/kkaefer/Code/gl/native/build/ios/Release-iphonesimulator/Settings.bundle
(1 failure)
make: *** [ipackage] Error 65
```

/cc @1ec5 @brunoabinader

--- a/platform/ios/ios.xcodeproj/project.pbxproj
+++ b/platform/ios/ios.xcodeproj/project.pbxproj
@@ -2268,6 +2268,7 @@
 		DA25D5BC1CCD9EDE00607828 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				INFOPLIST_FILE = framework/Settings.bundle/Root.plist;
 				PRODUCT_NAME = Settings;
 				SKIP_INSTALL = YES;
 				WRAPPER_EXTENSION = bundle;
@@ -2277,6 +2278,7 @@
 		DA25D5BD1CCD9EDE00607828 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				INFOPLIST_FILE = framework/Settings.bundle/Root.plist;
 				PRODUCT_NAME = Settings;
 				SKIP_INSTALL = YES;
 				WRAPPER_EXTENSION = bundle;

